README.md

```diff
@@ -10,45 +10,72 @@ See [open-telemetry/opentelemetry-java](https://github.com/open-telemetry/opente
+To use the library create a `DynatraceMetricExporter`.
+The `.getDefault()` method attempts to export to the [local OneAgent endpoint](https://www.dynatrace.com/support/help/how-to-use-dynatrace/metrics/metric-ingestion/ingestion-methods/local-api/).
 
-The Dynatrace exporter is added and set up like this:
+```java
+DynatraceMetricExporter exporter = DynatraceMetricExporter.getDefault();
+```
 
-First, create a `DynatraceMetricExporter`:
+Alternatively, or if no OneAgent is running on the host, the exporter can be set up using an endpoint URL and an API token with the "ingest metrics" (`metrics.ingest`) permission set.
+More information on setting up API access using tokens can be found [in the documentation](https://www.dynatrace.com/support/help/dynatrace-api/basics/dynatrace-api-authentication/) and [below](#dynatrace-api-token).
 
 ```java
 DynatraceMetricExporter exporter =
     DynatraceMetricExporter.builder()
-      .setApiToken({API_TOKEN})
-      .setUrl({INGEST_URL})
+      .setUrl("https://{your-environment-id}.live.dynatrace.com/api/v2/metrics/ingest")
+      .setApiToken({YOUR_API_TOKEN})
       .build();
 ```
 
-If the exporter is running on a host with an installed OneAgent, it can be initialized with:
+After acquiring a `DynatraceMetricExporter` object, it has to be registered using the `IntervalMetricReader` of OpenTelemetry:
 
 ```java
-DynatraceMetricExporter exporter = DynatraceMetricExporter.getDefault();
+// First, create a MetricsProducer (SdkMeterProvider is both a MeterProvider and a MetricsProducer):
+// This might not be the preferred way to acquire a MetricsProducer. Please refer to the OpenTelemetry documentation as the API is still in flux.
+SdkMeterProvider provider = SdkMeterProvider.builder().buildAndRegisterGlobal();
+
+// Then, tie together the exporter and the producer using an IntervalMetricReader:
+IntervalMetricReader.builder()
+    .setMetricProducers(Collections.singleton(provider))
+    .setExportIntervalMillis(60000)
+    .setMetricExporter(exporter)
+    .build()
+    .start();
 ```
 
-Then, register the exporter using the `IntervalMetricReader` of OpenTelemetry:
-
-```java
-IntervalMetricReader intervalMetricReader =
-    IntervalMetricReader.builder()
-        .setMetricProducers(
-            Collections.singleton(
-                OpenTelemetrySdk.getGlobalMeterProvider().getMetricProducer()))
-        .setExportIntervalMillis(5000)
-        .setMetricExporter(exporter)
-        .build();
-```
+The interval in which metrics are exported can be set on the `IntervalMetricReader` (see above).
+In the example case above, metrics are exported every 60 seconds.
 
 Once metrics are reported using the Metrics API, data will be exported to Dynatrace:
 
 ```java
-Meter meter = OpenTelemetry.getGlobalMeter("instrumentation-library-name","semver:1.0.0");
+Meter meter = GlobalMeterProvider.getMeter("opentelemetry.instrumented_library", "0.1.0-alpha");
```

Comment:
```suggestion
Meter meter = GlobalMeterProvider.getMeter("com.example.my_instrumentation_library", "0.1.0-alpha");
```
Note that this about the library providing instrumentation and not the library that is instrumented:
https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/api.md#get-a-meter
This is only the same if the app or library has instrumentation built-in already.